From c68f8bdcd4c58a0c1d92e20230a943e8089d865a Mon Sep 17 00:00:00 2001
From: John Crispin <blogic@openwrt.org>
Date: Thu, 10 Nov 2011 19:32:37 +0100
Subject: [PATCH 16/70] MAINTAINERS: add entry for Lantiq related files

Adds new entry to MAINTAINERS file for Lantiq SoC related code.

Signed-off-by: John Crispin <blogic@openwrt.org>
---
 MAINTAINERS |   12 ++++++++++++
 1 files changed, 12 insertions(+), 0 deletions(-)

--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -4353,6 +4353,18 @@ S:	Supported
 F:	Documentation/mips/
 F:	arch/mips/
 
+MIPS/LANTIQ
+M:	John Crispin <blogic@openwrt.org>
+M:	Thomas Langer <thomas.langer@lantiq.com>
+S:	Maintained
+F:	arch/mips/lantiq/*
+F:	drivers/i2c/busses/i2c-falcon.c 
+F:	drivers/mtd/maps/lantiq-flash.c
+F:	drivers/net/ethernet/lantiq_etop.c
+F:	drivers/spi/spi-falcon.c 
+F:	drivers/tty/serial/lantiq.c
+F:	drivers/watchdog/lantiq_wdt.c
+
 MISCELLANEOUS MCA-SUPPORT
 M:	James Bottomley <James.Bottomley@HansenPartnership.com>
 S:	Maintained
